integration-templates/anrok/actions/create-ephemeral-transaction.ts

```diff
@@ -0,0 +1,66 @@
+import type { NangoAction, TransactionActionResponse, SuccessTransaction, Transaction, AnrokTransactionData } from '../../models';
+
+import type { AnrokResponse } from '../types';
+import { mapFees } from '../mappers/fees.js';
+
+export default async function runAction(nango: NangoAction, rawInput: Transaction[]): Promise<TransactionActionResponse> {
+    const response: TransactionActionResponse = {
+        succeeded: [],
+        failed: []
+    };
+
+    const input = Array.isArray(rawInput) ? rawInput : [rawInput];
+
+    for (const transaction of input) {
+        const anrokTransaction: AnrokTransactionData = {
+            accountingDate: transaction.issuing_date,
+            currencyCode: transaction.currency,
+            customerId: transaction.contact.external_id,
+            customerName: transaction.contact.name,
+            customerAddress: {
+                line1: transaction.contact.address_line_1,
+                city: transaction.contact.city,
+                postalCode: transaction.contact.zip,
+                country: transaction.contact.country
+            },
+            lineItems: transaction.fees.map((fee) => ({
+                id: fee.item_id,
+                productExternalId: fee.item_code || '',
+                amount: fee.amount_cents || 0
+            }))
+        };
+
+        if (transaction.contact.taxable && transaction.contact.tax_number) {
+            anrokTransaction.customerTaxIds = [];
```

Comment:
why assigning an empty array and then pushing a value into it instead of assigning an array with the value?

Reply:
Unsure tbh, didn't write this part, but i'll update